recipes/gfakluge/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gfakluge" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/edawson/{{name}}.git
```

Comment:
It is recommended to use: url: https://github.com/edawson/gfakluge/archive/refs/tags/{{version}}.tar.gz

Reply:
If you need to use git_url you could add the following to pass linting step.

```
extra:
  skip-lints:
    - uses_vcs_url
    - missing_hash
```

Reply:
Using the git URL would be easiest, as upstream bundles a micro-library seemingly used only by this tool as a git submodule. Creating a whole new package just for that seems unnecessary, at least until someone wants it.